tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_pandas_filesystem_datasource.py

```diff
@@ -0,0 +1,71 @@
+import pathlib
+import great_expectations as gx
+import tempfile
+
+temp_dir = tempfile.TemporaryDirectory()
+full_path_to_project_directory = pathlib.Path(temp_dir.name).resolve()
+data_directory = pathlib.Path(
+    gx.__file__,
+    "..",
+    "..",
+    "tests",
+    "test_sets",
+    "taxi_yellow_tripdata_samples",
+    "first_3_files",
+).resolve(strict=True)
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_pandas_filesystem_datasource.py my_datasource">
+import great_expectations as gx
+
+context = gx.data_context.FileDataContext.create(full_path_to_project_directory)
```

Comment:
We may be able to update this to use `get_context()` instead of `.create()`

Reply:
@Shinnnyshinshin How can I use it -- does it allow me to specify the directory?

Reply:
🙇 Updated -- worked!